### PR TITLE
amend slight grammar issues & typos

### PR DESCRIPTION
**Pull request by scardozos**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -97,15 +97,15 @@ make html
 
 And you can open `docs/_build/html/index.html` to preview your works.
 
-> **Note:** Make sure you have install related modules. If you didn't, you can run `pip install -e .['doc']` to install all depedencies to build the documents.
+> **Note:** Make sure you have install related modules. If you don't, you can run `pip install -e .['doc']` to install all depedencies to build the documents.
```
Comment (sjsadowski) [inline CONTRIBUTING.md:100]: Can you fix the rest of the grammar on this line too?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -97,15 +97,15 @@ make html
...
 
 ## Make a Pull Request
 
 If you want to make a Pull Request, here are some suggestions:
 
 1. Check other pull requests first, make sure your contributions are not duplicate to others.
-2. Run all tests at your machine, make sure your contributions are works and does not break anything.
-3. Document and example will be very appreciated.
+2. Run all tests at your machine, make sure your contributions are working and do not break anything.
+3. Documents and examples will be very appreciated.
```
Comment (sjsadowski) [inline CONTRIBUTING.md:108]: Documents->Documentation

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ This project includes all static files of `swagger-ui` under `sanic_openapi/ui/`
 
 ## Code Style
 
-In `dev` requirements, it already including some modules related to code style:
+In `dev` requirements, it already includes some modules related to code style:
```
Comment (sjsadowski) [inline CONTRIBUTING.md:76]: suggested:
In `dev` requirements, some modules related to code style are already included:

**Review by sjsadowski — CHANGES_REQUESTED** [review]
Thanks for doing this, I hadn't noticed these issues previously. Take a look at the suggested changes and maybe do another quick review of grammar in the document.

**Comment by ahopkins** [comment]
I think I just copied and pasted from the main project. We should probably look at that one as well for changes.

**Comment by scardozos** [comment]
Hey, thanks for having a look :)

I made the changes requested, think all is okay now

**Review by sjsadowski — APPROVED** [review]

**Pull request merged**